test/e2e/resources/component-library/library/manifest

```diff
@@ -0,0 +1,7 @@
+title=FooLibrary
+subtitle=Tests brs support for ComponentLibrary instances
+major_version=1
+minor_version=0
+build_version=0
+sg_component_libs_provided=Foo
+hidden=1
```

Comment:
> The component library's manifest must include the rsg_version flag (rsg_version=1.2) to optimize channel performance.

It would be good to include the flag to make this more realistic

Reply:
Can do!  that's easy.